homeassistant/components/elkm1/__init__.py

```diff
@@ -175,14 +233,16 @@ def __init__(self, element, elk, elk_data):
         """Initialize the base of all Elk devices."""
         self._elk = elk
         self._element = element
+        self._prefix = elk_data['prefix']
         self._temperature_unit = elk_data['config']['temperature_unit']
         self._unique_id = 'elkm1_{}'.format(
+            elk.mac_address.lower() + '_' +
```

Comment:
Please only use new style string formatting, not string concatenation.

Reply:
ok